index.bs

```diff
@@ -853,3 +870,9 @@ The following APIs do not need modifications, because they will automatically fa
 - Firing of clipboard events, as well as {{Clipboard/read()|navigator.clipboard.read()}} and {{Clipboard/readText()|navigator.clipboard.readText()}} [[CLIPBOARD-APIS]]
 - {{Navigator/share()|navigator.share()}} [[WEB-SHARE]]
 - {{Element/requestPointerLock()|element.requestPointerLock()}} [[POINTERLOCK]]
+
+APIs that require the [=Document/visible=] [=Document/visibility state=]:
+- {{WakeLock/request()}} [[WAKE-LOCK]]
```

Comment:
```suggestion
- {{WakeLock/request()|navigator.wakeLock.request()}} [[WAKE-LOCK]]
```

Reply:
I made this change, also reverted to [SCREEN-WAKE-LOCK] which I didn't realize the original spec had instead of [WAKE-LOCK]. I don't know what source code that comes from.